components/brave_ads/core/internal/history/ad_history_manager.cc

```diff
@@ -144,11 +155,11 @@ mojom::UserReactionType AdHistoryManager::LikeAd(
     NotifyDidLikeAd(mutable_ad_history_item);
   }
 
-  return mutable_ad_history_item.ad_user_reaction_type;
+  std::move(callback).Run(/*success=*/true);
```

Comment:
This is a prerequisite for future work to transition from JSON to a database. That work will then return either true or false, instead of always true.